jena-arq/src/main/java/org/apache/jena/riot/tokens/TokenizerText.java

```diff
@@ -60,6 +61,26 @@
     private boolean finished = false ;
     private TokenChecker checker = null ;
 
+    // The code assumes that errors throw exception and so stop parsing.
+    private ErrorHandler errorHandler = new ErrorHandler() {
```

Comment:
NM, I see the mutator and accessor below. Still, the default instance could be factored out into a constant, I think.